src/current/_includes/v24.1/known-limitations/plpgsql-limitations.md

```diff
@@ -17,4 +17,10 @@
 - `NOT NULL` variable declarations are not supported. [#105243](https://github.com/cockroachdb/cockroach/issues/105243)
 - Cursors opened in PL/pgSQL execute their queries on opening, affecting performance and resource usage. [#111479](https://github.com/cockroachdb/cockroach/issues/111479)
 - Cursors in PL/pgSQL cannot be declared with arguments. [#117746](https://github.com/cockroachdb/cockroach/issues/117746)
-- `OPEN FOR EXECUTE` is not supported for opening cursors. [#117744](https://github.com/cockroachdb/cockroach/issues/117744)
\ No newline at end of file
+- `OPEN FOR EXECUTE` is not supported for opening cursors. [#117744](https://github.com/cockroachdb/cockroach/issues/117744)
+- The `print_strict_params` option is not supported in PL/pgSQL. [#123671](https://github.com/cockroachdb/cockroach/issues/123671)
+- The `FOUND` local variable, which checks whether a statement affected any rows, is not supported in Pl/pgSQL. [#122306](https://github.com/cockroachdb/cockroach/issues/122306)
+- By default, when a PL/pgSQL variable conflicts with a column name, CockroachDB resolves the ambiguity by treating it as a column reference rather than a variable reference. This behavior differs from PostgreSQL, where an ambiguous column error is reported, and it is possible to change the `plpgsql.variable_conflict` setting in order to prefer either columns or variables. [#115680](https://github.com/cockroachdb/cockroach/issues/115680)
+- When coercing a PL/pgSQL expression into a type that is too narrow, CockroachDB truncates values. This differs from PostgreSQL, where PL/pgSQL type coercion does not truncate the value. [#115385](https://github.com/cockroachdb/cockroach/issues/115385)
```

Comment:
Note: this is likely a Postgres bug, so maybe doesn't fit well as a CRDB "limitation". I'm not sure what the precedent is for a case like that, though.

Reply:
I agree it's a stretch. In that case, I feel we should remove the `docs-known-limitation` label from #115680 and not document it here. I'll go ahead and remove it from this PR.